source/styleguide/components.liquid

```diff
@@ -0,0 +1,127 @@
+---
+title: Components
+---
+
+<h2>Closed</h2>
+
+<dl class="sl-c-description-list sl-c-description-list--horizontal">
+  <div>
+    <dt>Dart Sass</dt>
+    <dd>✓</dd>
+  </div>
+  <div>
+    <dt>Ruby Sass</dt>
+    <dd>since 3.5.0</dd>
+  </div>
+  <div><a href="#">▶︎</a></div>
+</dl>
+
+<h2>Open</h2>
+
+<dl class="sl-c-description-list sl-c-description-list--horizontal">
+  <div>
+    <dt>Dart Sass</dt>
+    <dd>✓</dd>
+  </div>
+  <div>
+    <dt>Ruby Sass</dt>
+    <dd>since 3.5.0</dd>
+  </div>
+  <div><a href="#">▼</a></div>
+</dl>
+
+<div class="sl-c-callout">
+  <p>{% lorem 'paragraph' %}</p>
+</div>
+
+<h2>Alerts</h2>
+
+<div class="sl-c-alert">
+  <div class="sl-l-container">
+    <p>
+      <strong>Alert</strong>
+      {% lorem 'sentence' %}
+    </p>
+  </div>
+</div>
+
+<div class="sl-c-alert sl-c-alert--info">
+  <div class="sl-l-container">
+    <p>
+      <strong>Info Alert</strong>
+      {% lorem 'sentence' %}
+    </p>
+  </div>
+</div>
+
+<h2>Buttons</h2>
+
+<a href="#" class="sl-c-button">Link</a>
+<button class="sl-c-button" type="button">Button</button>
+
+<a href="#" class="sl-c-button sl-c-button--primary">Link</a>
+<button class="sl-c-button sl-c-button--primary" type="button">Button</button>
+
+<h2>Callouts</h2>
+
+<div class="sl-c-callout">
+  <h3>Callout</h3>
+  <p>{% lorem 'paragraph' %}</p>
+</div>
+
+<div class="sl-c-callout sl-c-callout--warning">
+  <h3>Warning</h3>
+  <p>{% lorem 'paragraph' %}</p>
+</div>
+
+<div class="sl-c-callout sl-c-callout--fun-fact">
+  <h3>Fun Fact</h3>
+  <p>{% lorem 'paragraph' %}</p>
+</div>
+
+<h2>Introduction</h2>
+
+<p class="sl-c-introduction">
+  {% lorem 'paragraph' %}
+</p>
+
+<div class="sl-c-introduction">
+  <p>{% lorem 'paragraph' %}</p>
+</div>
+
+<h2>Link Headers</h2>
+
+<h2 id="link-header-2">Link Header 2</h2>
+<h3 id="link-header-3">Link Header 3</h3>
+<h4 id="link-header-4">Link Header 4</h4>
+
+<h2>Lists</h2>
+
+<div class="sl-c-list-navigation-wrapper">
+{% markdown %}
+- [Vertical](#)
+- [Navigation](#)
+- [List](#)
+{% endmarkdown %}
+</div>
```

Comment:
This sticky nav thing isn't a new issue, but I wonder if they want to remove the sticky part on this page to just show the vertical list style?

![Screen Shot 2023-02-24 at 7 32 19 PM](https://user-images.githubusercontent.com/1581694/221331072-8108adad-cb83-4e1e-a7a9-7f5544010c87.jpg)

Reply:
Yeah, I agree that it's weird -- I'm fine with whatever you prefer here.

Reply:
I'll go ahead and merge this since it matches production, but feel free to clean this up in a later PR if you like.

Reply:
We can also just wrap it in an extra div here, and that will contain it's stickyness.